Makefile

```diff
@@ -78,16 +78,13 @@ ci-check: depends build check  ## Install dependencies and run all the tests.
 
 .PHONY: lint
 lint:
-	bzr ls-lint
+	flake8 --ignore E24,E121,E123,E125,E126,E221,E226,E266,E704 \
```

Comment:
minor issue: at least on xenial, python3-flake8 does not provide the /usr/bin/flake8 launcher. The flake8 package provides this.